proxy/http/HttpCacheSM.cc

```diff
@@ -171,7 +172,18 @@ HttpCacheSM::state_cache_open_write(int event, void *data)
     break;
 
   case CACHE_EVENT_OPEN_WRITE_FAILED:
-    if (open_write_tries <= master_sm->t_state.txn_conf->max_cache_open_write_retries) {
+    if (master_sm->t_state.txn_conf->cache_open_write_fail_action == HttpTransact::CACHE_WL_FAIL_ACTION_READ_RETRY) {
+      // fall back to open_read_tries
+      if (open_write_tries <= master_sm->t_state.txn_conf->max_cache_open_write_retries) {
+        Debug("http_cache", "[%" PRId64 "] [state_cache_open_write] cache open write failure %d. read retry triggered",
```

Comment:
What if `object_info != nullptr`. Does it disable the write retries even if the update case.

Reply:
Updated the docs and created a new PR. Closing this one. Thanks for the review!